bitnami/redis/README.md

```diff
@@ -302,6 +302,7 @@ The following table lists the configurable parameters of the Redis<sup>TM</sup>
 | `sysctlImage.mountHostSys`                            | Mount the host `/sys` folder to `/host-sys`                                                                                                         | `false`                                                 |
 | `sysctlImage.resources`                               | sysctlImage Init container CPU/Memory resource requests/limits                                                                                      | {}                                                      |
 | `podSecurityPolicy.create`                            | Specifies whether a PodSecurityPolicy should be created                                                                                             | `false`                                                 |
+| `podSecurityPolicy.enabled`                           | Enable PodSecurityPolicy                                                                                                                            | `false`                                                 |
```

Comment:
Can you move this above `.create`?

Also, could you update the `.create` description to mention that you also need the set `podSecurityPolicy.enabled` to true in order to enable it?